modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -437,6 +438,23 @@ const register = (editor: Editor): void => {
     default: false
   });
 
+  registerOption('disabled', {
```

Comment:
So can we name this `enabled` and default that to `true`? I mean we put a lot of effort getting rid of disabled though out the whole UI and options in previous work and at least I tried to continue that path. It can still be called isDisabled internally I think. At least then we don't have `disabled` in docs and other places publicly.

Reply:
Moved this to Jira.

Reply:
Thanks for raising this point, I've moved the discussion to Jira, I will update the ticket once I get inputs 🙂

Reply:
Where was the JIRA discussion? I've had a few days to think about it and I'm really tempted to agree. I was on the fence at first, but I like the way `editor.options.set('enabled', false)` reads.

Reply:
nevermind, found it

Reply:
just on the `editor.options.set('enabled', false)` part, in this particular case, if we call it `isDisabled` internally but using `enabled` publicly, this is an inverted logic (for us)?